### PR TITLE
 Add new version the function to check YouTube url

### PR DESCRIPTION
**Pull request by jdragonbae**
This patch adds new version the function to check YouTube url.

Refs: #776

### CODE REVIEW

**Comment by jdragonbae** [comment]
@romandev, @codeimpl : plz, suggest proper naming.

File: src/org/triple/banana/util/YouTubeUtil.java [hunk 1]
```diff
@@ -8,19 +8,24 @@
...
+import org.banana.cake.interfaces.BananaTab;
+import org.banana.cake.interfaces.BananaTabManager;
 
 import java.net.MalformedURLException;
 import java.net.URL;
 
 public class YouTubeUtil {
     private static final String TAG = "YouTubeUtil";
 
-    public static boolean isYouTubeDomainUrl(@NonNull String urlString) {
```
Comment (romandev) [inline src/org/triple/banana/util/YouTubeUtil.java:19]: This version should be kept.

File: src/org/triple/banana/media/MediaSuspendController.java [hunk 2]
```diff
@@ -16,7 +16,7 @@
             + "evt => evt.stopImmediatePropagation(), true);";
 
     public void DisableOnYouTube(BananaTab tab) {
-        if (YouTubeUtil.isYouTubeDomainUrl(tab.getUrl())) {
+        if (YouTubeUtil.isYouTubeUrl()) {
```
Comment (romandev) [inline src/org/triple/banana/media/MediaSuspendController.java:19]: Takes BananaTab here

File: src/org/triple/banana/util/YouTubeUtil.java [hunk 3]
```diff
@@ -8,19 +8,24 @@
...
+    public static boolean isYouTubeUrl() {
+        BananaTab tab = BananaTabManager.get().getActivityTab();
```
Comment (romandev) [inline src/org/triple/banana/util/YouTubeUtil.java:21]:
```
isYouTubeUrl() {
    isYouTubeUrl(tab);
}
```

**Review by romandev — COMMENTED** [review]
Please update commit description as well.

File: src/org/triple/banana/util/YouTubeUtil.java (hunk 3 above)
Comment (jdragonbae) [reply src/org/triple/banana/util/YouTubeUtil.java:21]: Done

File: src/org/triple/banana/media/MediaSuspendController.java (hunk 2 above)
Comment (jdragonbae) [reply src/org/triple/banana/media/MediaSuspendController.java]: Done

File: src/org/triple/banana/util/YouTubeUtil.java (hunk 1 above)
Comment (jdragonbae) [reply src/org/triple/banana/util/YouTubeUtil.java:19]: Thanks for the review. Done.

**Comment by jdragonbae** [comment]
> Please update commit description as well.

Thanks!

**Review by romandev — APPROVED** [review]
lgtm

**Pull request merged**